packages/kolibri-common/components/SidePanelModal/index.vue

```diff
@@ -20,27 +20,25 @@
 
           <!-- Fixed header -->
           <div
-            v-show="$slots.header"
             ref="fixedHeader"
             class="fixed-header"
             :style="fixedHeaderStyles"
           >
             <div class="header-content">
               <slot name="header">
               </slot>
+              <KIconButton
+                v-if="closeButtonIconType"
+                :icon="closeButtonIconType"
+                class="close-button"
+                :style="closeButtonStyle"
+                :ariaLabel="closeButtonMessage"
+                :tooltip="closeButtonMessage"
+                @click="closePanel"
+              />
```

Comment:
Moved this into here so that the rest of the stuff in `.header-content` is in the same container for spacing for the icon buttons.